### PR TITLE
Update create-bar-chart.md

### PR DESCRIPTION
**Pull request by kullJul**

### CODE REVIEW

**Comment by Court72** [comment]
@mberdugo

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: powerbi-docs/developer/visuals/create-bar-chart.md [hunk 1]
```diff
@@ -565,174 +533,58 @@ In addition to scaling, the update method also handles selections and colors. Th
...
     show = new formattingSettings.ToggleSwitch({
         name: "show",
         displayName: undefined,
         value: false,
-        topLevelToggle: true,
-        show: true
     });
 
-    // Formatting property `fill` color picker (formatting simple slice)
     fill = new formattingSettings.ColorPicker({
```
[the remark this answers is not in the document]
Comment (kullJul) [reply powerbi-docs/developer/visuals/create-bar-chart.md]: fixed 3e36673e81ac665c5a1047d587009e76c5b2caab

File: powerbi-docs/developer/visuals/create-bar-chart.md [hunk 2]
```diff
@@ -565,174 +533,58 @@ In addition to scaling, the update method also handles selections and colors. Th
...
-        show: true
+        value: { value: "#000000" }
     });
-
+    topLevelSlice = this.show;
     name: string = "enableAxis";
     displayName: string = "Enable Axis";
-    show: boolean = true;
-    slices: Array<FormattingSettingsSlice> = [this.show, this.fill];
+    slices = [this.fill];
```
[the remark this answers is not in the document]
Comment (kullJul) [reply powerbi-docs/developer/visuals/create-bar-chart.md]: fixed 3e36673e81ac665c5a1047d587009e76c5b2caab

File: powerbi-docs/developer/visuals/create-bar-chart.md [hunk 3]
```diff
@@ -565,174 +533,58 @@ In addition to scaling, the update method also handles selections and colors. Th
...
-    // Formatting property `fill` color picker (formatting simple slice)
     fill = new formattingSettings.ColorPicker({
         name: "fill",
         displayName: "Color",
-        value: { value: "#000000" },
-        show: true
+        value: { value: "#000000" }
     });
-
+    topLevelSlice = this.show;
```
[the remark this answers is not in the document]
Comment (kullJul) [reply powerbi-docs/developer/visuals/create-bar-chart.md]: fixed 3e36673e81ac665c5a1047d587009e76c5b2caab

**Review by MulyukovAidar — APPROVED** [review]

**Pull request merged**